<!DOCTYPE html>
<html lang="en" class="crsa-stop-animations">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="../../docs-assets/ico/favicon.png">

    <title>Explorations in Data Analyses for Metagenomic Advances in Microbial Ecology (EDAMAME) Workshop</title>

    <!-- Bootstrap core CSS -->
    <link href="css/simplex.bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/grid.css" rel="stylesheet">
</head>

<body>
    <nav class="navbar navbar-inverse" role="navigation">
        <div class="container-fluid">
            <div class="navbar-header">
                <a class="navbar-brand" href="http://edamame-course.org">EDAMAME</a>
            </div>
            <!-- nav links, etc., go here -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li><a href="#">Schedule (coming soon)</a>
                    </li>
                    <li><a href="#">Lesson Material (coming soon)</a>
                    </li>
                    <li><a href="#">Blog (coming soon)</a>
                    </li>
  		    <li><a href="https://edamame-course.github.io/website/logistics.html">Logistics</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container-fluid -->
    </nav>
    <div class="container">
        <div class="page-header">

            <h1>Explorations in Data Analyses for Metagenomic Advances in Microbial Ecology (EDAMAME) Workshop</h1>

        </div>
        <h4>August 13th to August 20th, 2014</h4>

        <h4><a href="http://www.kbs.msu.edu/">Kellogg Biological Station</a>, <a href="http://www.msu.edu">Michigan State University</a>, Hickory Corners, MI (<a href="https://www.google.com/maps/place/Kellogg+Biological+Station--Michigan+State+University/@42.405061,-85.400931,15z/data=!4m2!3m1!1s0x881788e3955439d3:0x5e86ae20e3a62103">map</a>)</h4>

        <h4><i><b>New! The EDAMAME Logistics Page is live!</b></i></h4>

        <h4>Instructors: Dr. Ashley Shade, Dr. Tracy Teal, Dr. Joshua Herr</h4>

        <h4>Tweet! #edamame2014</h4>

        <h4>Do you have questions?  Please e-mail us at <a href="mailto:edamame.course@gmail.com?Subject=EDAMAME%20Course" target="_top">edamame.course@gmail.com</a>.</h4>
        <h4>Thank you to Amazon Web Services for providing computing support for EDAMAME!</h4>
        <h4>Thank you to Michigan State's <a href="http://icer.msu.edu/hpcc">High Performance Computing Center</a> for providing coffee and snacks to EDAMAME!</h4>
        <br>

        <div class="row text-center">
            <img src="img/gull_lake.jpg" alt="gull_lake">
        </div>

        <h3>Course Description</h3>
        <p>This intensive one-week summer workshop will guide attendees in ecologically analyzing microbial communities that have been observed using short-read tag and shotgun metagenomics sequencing data.</p>
        <p></p>
        <p>No prior experience is necessary. Two years or more of graduate school in a biological science is <strong>strongly</strong> suggested. <strong>Faculty, postdocs, and research staff are more than welcome!</strong>
        </p>
        <p></p>
        <p>Students will gain practical experience in:</p>
        <ul>
            <li>Getting around in the shell - Introduction to Linux/Unix commands and executing Python scripts</li>
            <li>Tag-Sequence Analysis - Installing and using QIIME and mothur for processing and quality control of microbial tag-sequences, generating a taxon-table and for exploratory analyses</li>
            <li>Microbial metagenomes - data processing, quality control, gene assignment, and metagenome assembly</li>
            <li>R (vegan package) for multivariate analyses of microbial diversity and ecological interpretation</li>
            <li>Overview of online tools and databases with public microbial data: RDP, MG-RAST, CAMERA, VAMPS</li>
        </ul>

        <br>

        <div class="row text-center">
            <img src="img/class_picnic.jpg" alt="class_picnic">
        </div>



        <h3>Learning Objectives</h3>
        <p>Workshop attendees will be able to:</p>
        <ul>
            <li>Analyze microbial communities from an ecological perspective.</li>
            <li>Explain the process of high-throughput sequencing, provide an overview of data-handling specific to these technologies, and discuss their biases.</li>
            <li>Differentiate quantitative and categorical contextual data associated with microbial observations, and choose appropriate analysis strategies for each in interpreting community patterns. This includes: Visualizing community data, Testing for
                statistically significant patterns, Linking environmental or experimental treatment data to microbial data, Assess the contributions of individual taxa to community patterns.</li>
            <li>Develop a working proficiency with QIIME, mothur, and R, cite alternatives available, and identify resources for troubleshooting and further reading
            </li>
            <li>Become familiar with publicly accessible microbial sequence databases and the tools that they offer for deposition and analyses</li>
        </ul>

        <h3>Location, dates, and course structure</h3>
        <p>The course will be held at the W.K. Kellogg Biological Station on Gull Lake in western Michigan from 9 am on Wednesday, August 13rd through 5 pm on Wednesday, August 20th. Morning and afternoon lectures will be interspersed with practical hands-on
            tutorials. Tutorials will use existing datasets for demonstration, but students also will have opportunity to apply these tools to to their own datasets. The final day will include a Choose Your Own Adventure, during which students will opt
            into one of several available special interest topics, including “Networks analyses,” “Scripting in Python,” and “Working with markers other than 16S rRNA.”</p>
        <p>Room and board will be provided on-site (see enrolling, below). Sunday, August 17th, will be a half-day of rest &amp; relaxation. Students should expect to arrive at KBS by Wednesday, August 13rd at 9 am and stay at KBS through Wednesday, August
            20th at 5 pm. KBS is several hours away from the nearest airports and students coming from out of state should plan on arriving on Tuesday.</p>

        <h3>Applying for the course</h3>
        <p>An application is required, and we can accommodate only 24 attendees. We welcome people unaffiliated with MSU, and "students" of any career stage, including postdocs and faculty. Applicants are not selected on a first-come-first-serve basis, and
            we try to create a balanced class of diverse participants from disparate research areas.
        </p>
        <h3>Tuition, course cost, and enrolling</h3>
        <p>Course costs include room and board at KBS (see below: Lodging). It should be about $500. We will make arrangements for housing *after* choosing students; please do not contact KBS directly. Attendees are responsible for their own transportation
            to nearby airports; we will provide a shuttle from airport to KBS.</p>
        <div style="padding-left: 1em;">
            <h4>Lodging</h4>
            <p>For more information on course lodging and the course classroom, please see the Kellogg Biological Station website for information on the <a href="http://www.kbs.msu.edu/visit/conference-center/facilities">KBS Conference Facilities</a> and
                <a href="http://www.kbs.msu.edu/visit/conference-center/facilities/meeting-room-dimensions">KBS Meeting Rooms</a>. Once the the course roster is selected we will work with the selected students to make housing arrangements at KBS.</p>
            <h4>Taking the course for-credit</h4>
            <p>There is no formal option for taking the course for credit at MSU or elsewhere.</p>
            <h4>Supplies</h4>
            <p>Please plan to bring a laptop (Mac OS X, Windows, or Linux are all OK). We are not able to provide computers.</p>
            <h4>Lodging and family</h4>
            <p>We plan to have housing available for family groups, and expect to also make child care options available for children in the 3-10 age range.</p>
        </div>

        <h3>Instructors</h3>

        <img class="img-circle" src="img/ashley.jpg">

        <p><a href="http://ashley17061.wix.com/shadelab">Ashley Shade</a> holds a Ph.D. in Microbiology from the University of Wisconsin-Madison. She is currently an Assistant Professor at Michigan State Microbiology and Molecular Genetics, where her lab uses multidisciplinary
            tools to understand the ecology of microbial communities in soils, waters, and plants. She has particular interests in understanding temporal patterns of diversity, the role of rare taxa in driving community changes, and community responses
            to extreme disturbances. She has led workshops in microbial sequence analyses and ecological interpretation at Yale University and at the Canadian Society for Ecology and Evolution meeting.</p>

        <img class="img-circle" src="img/tracy.jpg">
        <p><a href="http://idyll.org/~tracyt/">Tracy Teal</a> received her PhD from California Institute of Technology and did her postdoctoral work at Michigan State University. She is currently an Assistant Professor at Michigan State University where she
            uses metagenomic approaches to assess the effects of land use change on microbial communities and associated greenhouse gas fluxes and works with the Enterics Research Investigational Network (ERIN) group to determine how the intestinal microbiota
            provides a barrier to incoming pathogens. She has developed several bioinformatics tools and is engaged in training researchers in computational techniques as an instructor for <a href="http://software-carpentry.org">Software Carpentry</a> and
            MSU’s <a href="http://icer.msu.edu">Institute for Cyber-Enabled Research</a>.</p>


        <img class="img-circle" src="img/josh.jpg">
        <p><a href="http://joshuaherr.com">Josh Herr</a> holds a Ph.D. in Integrative Biology from Pennsylvania State University. He is currently a post-doctoral researcher at Michigan State University in the Department of Microbiology &amp; Molecular Genetics.
            Josh’s research is focused on the taxonomic and functional identification of fungi and bacteria associated with soil, plants, insects, and -- through his association with the Enterics Research Investigational Network (ERIN) -- the intestinal
            microbiota of humans and cattle. Josh blogs about his research interests at <a href="http://cymeandcystidium.com">Cyme &amp; Cystidium</a>, is an instructor for <a href="http://software-carpentry.org">Software Carpentry</a>, and is an editor
            at the bioinformatics help forum <a href="http://www.biostars.org/u/1704">Biostar</a>.</p>

        <hr>

        <h3>Guest Speakers and Instructors</h3>
        <div class="col-md-6">
            <img class="img-circle" src="img/jack.jpg">
            <p><a href="http://www.bio.anl.gov/PI/gilbert.html">Jack Gilbert </a> University of Chicago, Argonne National Laboratories, Earth Microbiome Project</p>
            <img class="img-circle" src="img/schloss.jpg">
            <p><a href="http://www.mothur.org">Pat Schloss</a> University of Michigan, mothur Master</p>
            <img class="img-circle" src="img/stuart.jpg">
            <p><a href="http://www3.nd.edu/~sjones20/JonesUND/Home.html">Stuart Jones</a> University of Notre Dame, Aquatic Microbial &amp; Ecosystem Ecology</p>
            <img class="img-circle" src="img/cole.jpg">
            <p><a href="http://rdp.cme.msu.edu/">Jim Cole </a> Michigan State, Center for Microbial Ecology, Ribosomal Database Project</p>
            <p>Adina Chuang Howe</p>        
            <p>Kevin Theis, Michigan State</p> 
        </div>
        <div class="col-md-6">
            <img class="img-circle" src="img/kathryn.jpg">
            <p><a href="http://www.wmich.edu/biology/directory/docherty.html">Kathyrn Docherty</a> Western Michigan University, National Ecological Observatories Network (NEON)</p>
            <img class="img-circle" src="img/jay.jpg">
            <p><a href="http://www.indiana.edu/~microbes/">Jay Lennon</a> Indiana University</p>
            <img class="img-circle" src="img/titus.jpg">
            <p><a href="http://ged.msu.edu/">C. Titus Brown</a> Michigan State University, Computational Guru, Metagenomics Master</p>
            <img class="img-circle" src="img/tiedje.jpg">
            <p><a href="http://www.cme.msu.edu/">Jim Tiedje</a> Michigan State University, Center for Microbial Ecology</p>
            <p>Dirk Colbry, Michigan State, <a href="http://icer.msu.edu/hpcc">High Performance Computing Center</a></p>
            
        </div>
        <br>
        <hr>
        <div class="row top-buffer"></div>
        <h4>More guest speakers to be confirmed...</h4>
        <br>

        <div class="row text-center">
            <img src="img/carriage_house.jpg" alt="carriage_house">
        </div>

        <br>

        <!-- /container -->
        <!-- Bootstrap core JavaScript
    ================================================== -->
        <!-- Placed at the end of the document so the pages load faster -->
        <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
        <script src="js/bootstrap.min.js"></script>
    </div>



</body>

</html>
